composer.json

```
@@ -19,39 +19,41 @@
         "erusev/parsedown": "^1.6",
         "sensio/framework-extra-bundle": "^6.1",
         "symfony/apache-pack": "^1.0",
-        "symfony/asset": "^5.3",
-        "symfony/console": "^5.3",
-        "symfony/dotenv": "^5.3",
-        "symfony/expression-language": "^5.3",
-        "symfony/flex": "^1.1",
-        "symfony/form": "^5.3",
-        "symfony/framework-bundle": "^5.3",
-        "symfony/intl": "^5.3",
-        "symfony/mailer": "^5.3",
+        "symfony/asset": "^5.4.0-beta1",
```

I think, you can change all of those to…

```suggestion
        "symfony/asset": "^5.4",
```